docs/source/metrics.rst

```diff
@@ -35,7 +35,7 @@ event. It computes the metric based on its accumulated states. The metric value
     print(f"Accuracy: {state.metrics['accuracy']}")
 
 
-If the engine's output is not in the format ``(y_pred, y)`` or ``{'y_pred': y_pred, 'y': y, ...}``, the user can
+If the engine's output is not in the format ``(y_pred, y)``, the user can
```

Comment:
Here, it is fine to output dict of y_pred and y. Please revert this as well

Reply:
I think the code example right below it suggest that even if the engine's output is already a dict``{'y_pred': y_pred, 'y_true': y, ...}`` returned by process_function, it is still needed to pass output_transform to Accuracy and convert this dict into a tuple``(y_pred, y)``. Please let me know and my apologies if I were wrong.